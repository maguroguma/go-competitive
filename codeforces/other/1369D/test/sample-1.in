7
1
2
3
4
5
100
2000000
