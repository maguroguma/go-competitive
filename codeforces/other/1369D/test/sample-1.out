0
0
4
4
12
990998587
804665184
